packages/framework-provider-azure/src/helpers/query-helper.ts

```diff
@@ -171,8 +174,10 @@ function buildAttributeValue(
           value: element,
         })
       })
-    } else if (typeof value === 'object' && key !== 'includes') {
+    } else if (typeof value === 'object' && key !== 'includes' && value !== null) {
       attributeValues = [...attributeValues, ...buildExpressionAttributeValues({ [key]: value }, usedPlaceholders)]
+    } else if (key === 'isDefined') {
+      // skip this parameter
     } else {
```

Comment:
Leaving this block with no code here seems a bit weird, maybe...

```suggestion
    } else if (key !== 'isDefined') {
```